README.md

```diff
@@ -39,31 +96,42 @@ You can pass additional arguments to Theia after the image name, for example to
 
     # Linux, macOS, or PowerShell
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "$(pwd):/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
-    
+
     # Windows (cmd.exe)
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "%cd%:/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
 
-### Image Variants
+### Electron Apps
 
-#### `theiaide/theia:latest`
+<img
+    src='./assets/electron.png'
+    alt='electron logo'
+    width="50px"
+/>
 
-This image is based on the latest stable released version.
+| Application Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-cpp-electron` | Theia-based C/C++ desktop IDE | [docs](./theia-cpp-electron/README.md) |
+| `theia-electron` | Theia-based JavaScript/TypeScript (Web Technologies) desktop IDE | [docs](./theia-electron/README.md) |
 
-#### `theiaide/theia:<version>`
+---
 
-This image is based on the given stable released version.
+<br />
 
-#### `theiaide/theia:next`
+## Plugins
 
-This image is based on the nightly published version.
+Theia supports VS Code extensions as plugins. If a given image has plugin support, one can
+include their own plugins under the user's home: `/home/theia/plugins`.
 
-#### `theiaide/theia:<version>-next.<commit hash>`
+## Tips & Tricks
 
-This image is based on the given nightly published version.
+- #### Build Options:
+    - `--build-arg strip=true` strips the application to save space but with reduced debuggability.
```

Comment:
Grepping quickly I only saw this build option supported in `theia-cpp-docker'. Maybe that should be moved to that specific README?